code/game/objects/items/toys.dm

```diff
@@ -904,6 +917,210 @@
 		'sound/weapons/cablecuff.ogg' = 1,
 	)
 
+/obj/item/toy/plushie/skrellplushie
+	name = "skrell plushie"
+	desc = "The latest from 'SoftSkrells.net', features its very own headpocket! Warble!"
+	icon_state = "plushie_skrell"
+	rare_hug_sound = 'sound/effects/warble.ogg'
+	rare_hug_word = "Warble!"
+	var/obj/item/headpocket_item
+
+/obj/item/toy/plushie/skrellplushie/examine(mob/user)
+	. = ..()
+	. += "<span class='notice'>Alt-click to put something small inside the headpocket, or take an item out.</span>"
+
+/obj/item/toy/plushie/skrellplushie/AltClick(mob/user)
+	if(!Adjacent(user) || HAS_TRAIT(user, TRAIT_HANDS_BLOCKED))
+		return
+	var/obj/item/I = user.get_active_hand()
+	if(I == src)
+		return
+	if(!I)
+		if(!headpocket_item)
+			return
+		to_chat(user, "<span class='notice'>You remove [headpocket_item] from [src].</span>")
+		headpocket_item.forceMove(get_turf(src))
+		user.put_in_hands(headpocket_item)
+		headpocket_item = null
+		return
+	if(I.w_class > WEIGHT_CLASS_SMALL)
+		to_chat(user, "<span class='warning'>You cannot fit [I] in [src]!</span>")
+		return
+	if(!iscarbon(user))
+		return
+	if(headpocket_item)
+		to_chat(user, "<span class='warning'>[src] already has an item in its headpocket!</span>")
+		return
+	if(!user.drop_item())
+		to_chat(user, "<span class='warning'>You cannot slip [I] inside [src]!</span>")
+		return
+	user.visible_message("<span class='notice'>[user] places [I] into [src].</span>", "<span class='notice'>You place [I] into [src].</span>")
+	add_fingerprint(user)
+	I.forceMove(src)
+	headpocket_item = I
+
+/obj/item/toy/plushie/skrellplushie/Destroy()
+	headpocket_item.forceMove(get_turf(src))
+	headpocket_item = null
+	return ..()
+
+/obj/item/toy/plushie/humanplushie
+	name = "human plushie"
+	desc = "This plushie is slightly less popular than its counterparts. The designers obviously didn't find humans that endearing..."
+	icon_state = "plushie_human"
+	poof_sound = list('sound/weapons/thudswoosh.ogg' = 30,
+					'sound/goonstation/voice/male_scream.ogg' = 1)
+
+/obj/item/toy/plushie/borgplushie
+	name = "borg plushie"
+	desc = "The synthetic backbone of the station, rendered in plush form. Inbuilt flashlight included!"
+	icon_state = "plushie_borg"
+	var/borg_plushie_overlay
+	var/on = FALSE
+
+/obj/item/toy/plushie/borgplushie/Initialize(mapload)
+	. = ..()
+	borg_plushie_overlay = (pick("plushie_borgjan", "plushie_borgsec", "plushie_borgmed", "plushie_borgmine", "plushie_borgserv", "plushie_borgassist", "plushie_borgengi"))
+	update_icon()
+
+/obj/item/toy/plushie/borgplushie/attack_self(mob/user)
+	on = !on
+	update_brightness()
+	return ..()
+
+/obj/item/toy/plushie/borgplushie/proc/update_brightness()
+	if(on)
+		set_light(4)
+	else
+		set_light(0)
+	update_icon()
```

Comment:
You'll want to call `update_appearance()` (with some flags) instead of invoking update_icon directly.